src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/"
+        class="logo"
+      >
+        <img
+          src="images/moyo_1.png"
+          alt="Logo"
+          class="logo-image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="ul__nav">
+          <li class="li_nav">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="li_nav">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="li_nav">
```